### ISSUE TITLE
MaxQuant Data vs proteoDA Tutorial Example data

### ISSUE THREAD

**Issue by devenderarora** [ByrumLab/proteoDA#247]
Dear Team,
I am working with proteomics data and have implemented proteoDA on data generated using MaxQuant. However, when I examined the example data provided in the tutorial, I found that the abundance of proteins observed was much higher than what we obtained from MaxQuant. Could you provide information on how the test data was generated or its source? I tried to find this information in the manuscript but was unsuccessful.

Regards,
Devender

**Comment by ByrumLab** [comment]
Hi Devender,

The example data is from a DIA run using MS2 exclusive intensities instead of DDA. 

Best,
Stephanie

**Comment by devenderarora** [comment]
Thankyou for the prompt response and clarifying the issue. We also have few more issue while doing the analysis. While reproducing the test data we observed that the intragroup variability and correlation does not correspond to what shown in example file at:
https://github.com/ByrumLab/proteoDA/blob/main/vignettes/tutorial.html

we followed the same example and our results shows:

![image](https://github.com/ByrumLab/proteoDA/assets/56953623/723cd68d-e6b7-4df2-a1e0-d067624e5396)

and MAplots
![image](https://github.com/ByrumLab/proteoDA/assets/56953623/8163193f-4c43-4eaa-90dd-8889565e3096)

Can you please guide us and share how "cycloess" was chosen as based on our interpretation we found quantile normalization represent better flat orange line in MA plot.

Regards,
Devender

**Comment by tjthurman** [comment]
Hi @devenderarora,

Thanks for catching the issue with the vignette. The plot you generated is correct, the pre-built vignette on GitHub was out-of-date: I've submitted a PR to fix it. As for picking a normalization method, for the tutorial we picked cyclic loess as a demonstration (it does have a slightly better intragroup correlation, PCV, and PMAD), but quantile normalization would be a good choice too. Generally, it's a good thing if the different normalization methods have similar metrics, as they likely will produce similar results. 

Best,
Tim
